src/doc/src/reference/config.md

```diff
@@ -1257,6 +1257,19 @@ Controls whether or not extra detailed messages are displayed by Cargo.
 Specifying the `--quiet` flag will override and disable verbose output.
 Specifying the `--verbose` flag will override and force verbose output.
 
+#### `term.warnings`
```

Comment:
> Where should the config option go?

Reply:
`term` feels wrong.

We have `build` but this will control other circumstances (maybe even `cargo metadata`!).  However, `build` might be the first place people look.

Maybe we should have some kind of general category?